cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -64,7 +66,22 @@ MATCH (n:User) WHERE n.name CONTAINS 'ndy' RETURN n.name;
 
 This query will perform several checks:
```

Comment:
```suggestion
This query performs several checks:
```
I know this is not part of what you have changed, but we strive to keep the active voice and present tense.